lettermgmt/models/letter_type.py

```diff
@@ -20,18 +20,16 @@
 #    along with this program.  If not, see http://www.gnu.org/licenses/.
 #
 ##############################################################################
-
 from openerp.osv import fields, orm
-from openerp.tools.translate import _
 
 
 class letter_type(orm.Model):
     """Class to define various types for letters like : envelope,parcel,
     etc."""
     _name = 'letter.type'
-    _description = _("Letter Type")
+    _description = "Letter Type"
     _columns = {
-        'name': fields.char('Type', size=32, required=True),
-        'code': fields.char('Code', size=8, required=True),
+        'name': fields.char('Type', required=True),
+        'code': fields.char('Code', required=True),
     }
     _sql_constraints = [('code_uniq', 'unique(code)', 'Code must be unique !')]
```

Comment:
> Add _('Code must be unique !') to be translate.

I am sure this is not required, it is added to the translations by the ORM when it looks up the _sql_constraints.